### PR TITLE
Add note at top of each tutorial to show which version of SnarkyJS it was tested with

### LINKED ISSUE #205: Add note at top of each tutorial: `Tested with SnarkyJS v#.#.#`
So readers will know if SnarkyJS' version advances and a tutorial still needs to be updated. We could do this for the `0.8.0` SnarkyJS release.

CC @es92 for visibility

### PR DESCRIPTION
**Pull request by ymekuria**
Closes #205 

A note was added at the top of each tutorial stating which SnarkyJS version it was tested with.

Screen Shot 2023-02-01 at 2 38 35 AM

### CODE REVIEW

File: examples/zkapps/04-zkapp-browser-ui/ui/pages/_app.page.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default function App() {
         console.log('zkApp compiled');
 
         const zkappPublicKey = PublicKey.fromBase58(
-          'B62qph2VodgSo5NKn9gZta5BHNxppgZMDUihf1g7mXreL4uPJFXDGDA'
```
Comment (jasongitmail) [inline examples/zkapps/04-zkapp-browser-ui/ui/pages/_app.page.tsx:62]: @ymekuria Is this intentional to be included?

Comment (ymekuria) [reply examples/zkapps/04-zkapp-browser-ui/ui/pages/_app.page.tsx:62]: @jasongitmail, I added an updated zkApp public key of a deployed contract for this tutorial. It is unrelated to adding notes to each tutorial and can be removed here and added to another PR.

Comment (jasongitmail) [reply examples/zkapps/04-zkapp-browser-ui/ui/pages/_app.page.tsx:62]: Ok, for expediency will merge

**Review by jasongitmail — APPROVED** [review]
Thanks!

**Pull request merged**